ironfish/src/rpc/routes/wallet/sendTransaction.ts

```diff
@@ -64,6 +65,7 @@ router.register<typeof SendTransactionRequestSchema, SendTransactionResponse>(
   `${ApiNamespace.wallet}/sendTransaction`,
   SendTransactionRequestSchema,
   async (request, node): Promise<void> => {
+    request.data.outputs.map((o) => ValidationUtils.assertValidMemo(o.memo))
```

Comment:
Instead of introducing these custom validation functions, it would probably be easier to change line 43 to `memo: yup.string().defined().max(32)`

You can apply this to all the changes in this PR I think.

Reply:
Oh cool, I didn't know yup could do that. Yea this is much better!